src/index.ts

```diff
@@ -1,3 +1,10 @@
-import comparePdf from './compare.pdf';
+import { comparePdf } from './comparePdf.js';
 export default comparePdf;
-export * from './types';
+
+// export other libraries types that this project use
+export { PdfToPngOptions } from 'pdf-to-png-converter';
+export { Area, Color } from 'png-visual-compare';
```

Comment:
not sure that this is necessary... why i should expose types that not belong to my library?